src/Terrain/TerrainType.java

```diff
@@ -0,0 +1,123 @@
+package Terrain;
+
+import java.awt.Color;
+
+public class TerrainType
```

Comment:
FWIW, this is a lot more readable than I thought it would be, so props for that.